app/classifier/drawing-tools/ellipse.cjsx

```diff
@@ -53,6 +56,13 @@ module.exports = React.createClass
     x: @props.mark.rx * Math.cos theta
     y: -1 * @props.mark.ry * Math.sin theta
 
+  dragEnd: (target) ->
+    @context.geordi?.logEvent type: 'drag-ellipse'
+    deleteIfOutOfBounds target
+
+  logResize: ->
+    @context.geordi?.logEvent type: 'resize-ellipse'
+
```

Comment:
This was actually added according to the documentation we received as to what was to be logged in the classification interface. Looks like the drawing events we're currently targeting are draw, drag, resize, delete, select-option, and task-end.